auctions/tests/model_tests.py

```diff
@@ -273,3 +289,14 @@ def test_save_updates_datetimes(self):
                         "Vote.modified should be auto-populated on update.")
         self.assertTrue(datetime.now() >= test_vote_modified,
                         "Vote.modified should be auto-populated.")
+
+    def test_user_claim_unique(self):
+        mommy.make(Vote, claim=self.claim, user=self.user1, approved=True)
+        with self.assertRaises(IntegrityError):
```

Comment:
ooo, nice use of `with` ... I should change some of my `try ... catch IntegrityError` type assertions with this.